gremlin-python/src/main/jython/tests/conftest.py

```diff
@@ -1,4 +1,4 @@
-'''
+"""
```

Comment:
any reason you switched to double quotes here? seems like we used singles everywhere else.

Reply:
When possible, I try to follow the style guidelines laid out in [PEP 8](https://www.python.org/dev/peps/pep-0008/#string-quotes). Docstrings specifically relate to  [PEP 257](https://www.python.org/dev/peps/pep-0257/). Gremlin Python wasn't originally written with these guidelines in mind, so sometimes I make little style fixes in code that I am updating. Maybe not the best idea, since it doesn't really relate to the PR.

Reply:
i'm fine with adhering to the style guides. perhaps just do it consistently? perhaps CTR in fixes across the board for all the license headers, so that it's not just in this one file.

Reply:
Ok willdo.